apps/settings.gradle.kts

```diff
@@ -20,5 +20,6 @@ include("http-api-app")
 include("seed-node-app")
 include("oracle-node-app")
 include("node-monitor-web-app")
+include("chatterbox-app")
```

Comment:
Maybe better name it more generic for stress testing the network.  There might be more tools in future, like simulation tools. So maybe the module should be like `network-lab`? There could be also future integration tests/test harness or other stuff... In that sense its better to make the app also more generic and use config entries to define which particular stresstest/simutation you want to run. 
The dev can then pass the jvm arg to run a particular scenario.

The changes in config and app can be also done in a separate PR, but the module rename would be good to get changed here.